src/Adapter/Product/AdminProductDataProvider.php

```diff
@@ -318,28 +323,28 @@ public function getCatalogProductList($offset, $limit, $orderBy, $sortOrder, $po
         ));
 
         $sql = $this->compileSqlQuery($sqlSelect, $sqlTable, $sqlWhere, $sqlOrder, $sqlLimit);
-        $products = \Db::getInstance()->executeS($sql, true, false);
-        $total = \Db::getInstance()->executeS('SELECT FOUND_ROWS();', true, false);
+        $products = Db::getInstance()->executeS($sql, true, false);
+        $total = Db::getInstance()->executeS('SELECT FOUND_ROWS();', true, false);
         $total = $total[0]['FOUND_ROWS()'];
 
         // post treatment
-        $currency = new \CurrencyCore(\Configuration::get('PS_CURRENCY_DEFAULT'));
+        $currency = new (\Configuration::get('PS_CURRENCY_DEFAULT'));
         foreach ($products as &$product) {
             $product['total'] = $total; // total product count (filtered)
-            $product['price_final'] = \ProductCore::getPriceStatic($product['id_product'], true, null,
+            $product['price_final'] = Product::getPriceStatic($product['id_product'], true, null,
                 (int) \Configuration::get('PS_PRICE_DISPLAY_PRECISION'), null, false, true, 1,
```

Comment:
Several occurrence of `\Configuration` in the file, don't know if you want to replace them ?